exec/impl/src/main/java/org/jboss/windup/exec/RuleUtils.java

```diff
@@ -18,24 +18,30 @@
      */
     public static String prettyPrintRule(Rule rule)
     {
-        StringBuilder builder = new StringBuilder();
+        StringBuilder sb = new StringBuilder();
```

Comment:
(This one is less relevant/important. StringBuilder is one of those cases where a short variable can be good.)